node/src/isolate/external.cpp

```diff
@@ -159,6 +160,22 @@ struct spawn_load_t :
         cancelled(false)
     {}
 
+    spawn_load_t(std::weak_ptr<external_t::inner_t> _inner,
```

Comment:
Both constructors are needed?
May be we can use default arg for `extended_info` in causing code.

Reply:
remove code for spawn part, much simplified now